addons/l10n_cz/data/tax_report.xml

```diff
@@ -0,0 +1,819 @@
+        <field name="filter_multi_company">disabled</field>
+        <field name="default_opening_date_filter">last_month</field>
+        <field name="filter_date_range" eval="True"/>
+        <field name="filter_period_comparison" eval="True"/>
+        <field name="filter_show_draft" eval="True"/>
+        <field name="column_ids">
+            <record id="l10n_cz_vat_declaration_tax_base" model="account.report.column">
+                <field name="name">Tax base</field>
+                <field name="expression_label">tax_base</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_output_tax" model="account.report.column">
+                <field name="name">Output Tax</field>
+                <field name="expression_label">output_tax</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_value" model="account.report.column">
+                <field name="name">Value</field>
+                <field name="expression_label">value</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_in_full_amount" model="account.report.column">
+                <field name="name">In full</field>
+                <field name="expression_label">in_full_amount</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_reduced_claim" model="account.report.column">
+                <field name="name">Short deduction</field>
+                <field name="expression_label">reduced_claim</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_coefficient" model="account.report.column">
+                <field name="name">Coefficient</field>
+                <field name="expression_label">coefficient</field>
+                <field name="figure_type">percentage</field>
+            </record>
+        </field>
+        <field name="line_ids">
+            <record id="l10n_cz_vat_declaration_line_1" model="account.report.line">
+                <field name="name">I. Taxable transactions</field>
+                <field name="hierarchy_level">0</field>
+                <field name="children_ids">
+                    <record id="l10n_cz_vat_declaration_line_2" model="account.report.line">
+                        <field name="name">Supply of goods or services with a place of performance in the domestic territory</field>
+                        <field name="children_ids">
+                            <record id="l10n_cz_vat_declaration_line_3" model="account.report.line">
+                                <field name="name">Basic - line 1</field>
+                                <field name="code">R1</field>
```

Comment:
Wouldn't it make sense to keep the code related to the xml report (`cz_tax_report_template`)?
Or is this an official naming?

Reply:
@william-andre This is the code indicated on the partner provided excel sheet https://docs.google.com/spreadsheets/d/1xqDqpLwY5nCyNA5WuzLSPdR52Jq6f8IPxU4S3Bh1Nw4/edit?pli=1&gid=1709389226#gid=1709389226